App/VSTS/Save.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { Provider, connect } from 'react-redux';
+// import { changeSave, StageEnum } from '../Reducers/ActionsET';
+import { Rest } from '../RestHelpers/rest';
+import { changeStage, Stage } from '../Reducers/ActionsET';
+import { IWorkItem } from '../Reducers/ReducersET';
+ /**
+  * Represents the Save Properties
+  * @interface ISaveProps
+  */
+export interface ISaveProps {
+    /**
+     * dispatch to map dispatch to props
+     * @type {any}
+     */
+    dispatch?: any;
+    /**
+     * the entire work item property
+     * @type {IWorkItem}
+     */
+    workItem?: IWorkItem;
+}
+/**
+ * Maps elements of the state to properties
+ * @returns {ISaveProps}
+ * @param {any} state
+ */
+function mapStateToProps (state: any): ISaveProps  {
+      return { workItem: state.workItem };
+}
+
+@connect (mapStateToProps)
+/**
+ * Renders the Save button and makes REST api calls
+ * @class { Save }
+ */
+export class Save extends React.Component<ISaveProps, {}> {
+/**
+ * States whether to disable the save button or not
+ * @type {boolean}
+ */
+   public isDisabled: boolean = false;
+/**
+ * Dispatches the action to change the Stage and make the REST call to create the work item
+ * @returns {void}
+ */
+   public handleSave(): void {
+      this.props.dispatch(changeStage(Stage.Saved));
+      Rest.createWorkItem ('t-emtenc@microsoft.com', 'o365exchange', 'Outlook Services/Ecosystem - Ext VSTS', 'Bug',
```

Comment:
These values are hard-coded until I get the account, project, and area from Miranda's part of the store.